README.md

```diff
@@ -1,2 +1,4 @@
 # HyperBlog
-Un blog increible para el curso de git y github de plazti 
+Un blog increible para el curso de git y github de plazti
+
+MrJuanGaviriaK
```

Comment:
El nombre correcto debe ser Luisa Franco

Reply:
I'm going to check! thanks